src/catalog/src/builtin.rs

```diff
@@ -8359,6 +8661,7 @@ pub static BUILTINS_STATIC: LazyLock<Vec<Builtin<NameReference>>> = LazyLock::ne
         Builtin::View(&MZ_COMPUTE_EXPORTS),
         Builtin::View(&MZ_DATAFLOW_ARRANGEMENT_SIZES),
         Builtin::View(&MZ_EXPECTED_GROUP_SIZE_ADVICE),
+        Builtin::View(&MZ_INDEX_ADVICE),
```

Comment:
Make sure that this is added in the list AFTER any other object it may depend on.